Tools/apiCompat/baseline/ApiCompatBaseline.net45.txt

```diff
@@ -1,2 +1,7 @@
+CannotRemoveAttribute : Attribute 'Newtonsoft.Json.JsonObjectAttribute' exists on 'Microsoft.IdentityModel.Tokens.JsonWebKey' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'Newtonsoft.Json.JsonObjectAttribute' exists on 'Microsoft.IdentityModel.Tokens.JsonWebKeySet' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'Newtonsoft.Json.JsonObjectAttribute' exists on 'Microsoft.IdentityModel.Protocols.OpenIdConnect.OpenIdConnectConfiguration' in the contract but not the implementation.
+MembersMustExist : Member 'Microsoft.IdentityModel.Protocols.OpenIdConnect.OpenIdConnectMessage..ctor(Newtonsoft.Json.Linq.JObject)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'Microsoft.IdentityModel.Tokens.JsonWebKeySet..ctor(System.String, Newtonsoft.Json.JsonSerializerSettings)' does not exist in the implementation but it does exist in the contract.
```

Comment:
assuming this is for compatibility with newtonsoft serialization?

Reply:
We removed public ctors that take JObject/JsonSerializerSettings. As a result of that, ApiCompat will fail and report compatibility issues.  We are _allowing_ these issues by placing the exact error message reported into a baseline file.
I'm not sure if I understood the need for the test in this scenario.